lib/postgres_v3_experimental.dart

```diff
@@ -247,7 +249,7 @@ class PgSessionSettings {
     this.connectTimeout,
     this.queryTimeout,
     this.timeZone,
-    this.encoding,
+    required this.encoding,
```

Comment:
This shouldn't be a `required` setting. Let's make `encoding` nullable. (Also helps with subclassing in other packages that rely on this).

Reply:
done,
I think it should be non-null but initializing with utf8, but I followed your recommendation and put it as nullable